learning/courses/quantum-computing-in-practice/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Overview
+description: A free IBM course on utility-scale quantum computing featuring realistic use cases and best practices
+---
+
+# Overview
+
+This course focuses on today's quantum computers and how to use them to their full potential. It covers realistic potential use cases for quantum computing as well as best practices for running and experimenting with quantum processors having 100 or more qubits.
+
+## Recommended background
+
+To make the most out of this course, we recommend a familiarity with quantum information and quantum circuits, at the level explained in the [Basics of quantum information course](https://learning.quantum.ibm.com/course/basics-of-quantum-information) on this platform.
```

Comment:
This URL makes sense to start out with. We'll want to remember to scrub all links to `learning.quantum.ibm.com` before going live.